automates/text_reading/src/main/scala/org/clulab/aske/automates/OdinActions.scala

```diff
@@ -357,6 +363,60 @@ class OdinActions(val taxonomy: Taxonomy, expansionHandler: Option[ExpansionHand
     resolved ++ woIt
   }
 
+  def resolveModelCoref(mentions: Seq[Mention], state: State = new State()): Seq[Mention] = {
+    val (models, nonModels) = mentions.partition(m => m.label == "ModelDescr")
+    val (theModel, modelNames) = models.partition(m => m.arguments.contains("model") && m.arguments("model").head.foundBy == "the/this_model" || m.arguments.contains("model") && m.arguments("model").head.foundBy == "model_pronouns")
```

Comment:
does this cover "It"?

Reply:
Yes, "It" (or "it") is captured by "model_pronouns" rule.